src/libraries/System.Private.Xml/src/System/Xml/Schema/XsdDateTime.cs

```diff
@@ -495,7 +497,17 @@ InternalKind switch
         /// </summary>
         public override string ToString()
         {
-            var vsb = new ValueStringBuilder(stackalloc char[64]);
+            Span<char> destination = stackalloc char[CharStackBufferSize];
```

Comment:
Why did we remove ValueStringBuilder?

Reply:
@krwq because now I am using TryFormat method to write into the destination formatted value.